src/decisionengine/framework/engine/Workers.py

```diff
@@ -66,8 +66,8 @@ def run(self):
         # so we can reuse it here and reduce duplication
         logger_rotate_by = self.logger_config.get("file_rotate_by", "size")
 
-        logconf = copy.deepcopy(configDict.pylogconfig)
-
+        #logconf = copy.deepcopy(configDict.pylogconfig)
+        logconf = configDict.pylogconfig
```

Comment:
I'm pretty sure the `import copy` and `copy.deepcopy` are actually required here...

Reply:
oops, I was playing around and forgot to change it back. Will fix.

Reply:
Pat and I reached an agreement on the logging config dict. I defined the dict to be global in logging_configDict.py, which is how it has been defined in my head. WIth this, none of the copying, etc is needed.